asm/macros/map.inc

```diff
@@ -22,9 +22,8 @@
 	@ Defines an object event template for map data, to be used by a normal object. Mirrors the struct layout of ObjectEventTemplate in include/global.fieldmap.h
 	.macro object_event index:req, gfx:req, x:req, y:req, elevation:req, movement_type:req, x_radius:req, y_radius:req, trainer_type:req, sight_radius_tree_etc:req, script:req, event_flag:req
 	.byte \index
-	.byte \gfx
+	.2byte \gfx
```

Comment:
Isn't this related to overworld expansion? If so, aren't there way more `byte` to `2byte` changes that need to happen across the repo?

Reply:
Upon closer inspection at other files, it does seem like overworld expansion is currently a part of this PR. I'm personally fine with it (I don't know if followers work without it, after all), but we should document that it's a part of this branch, too

Reply:
It is in the PR description 👀

Reply:
I’m just blind, apologies